src/modules/registrypreview/RegistryPreviewUI/MainWindow.Utilities.cs

```diff
@@ -428,14 +430,39 @@ private bool ParseRegistryFile(string filenameText)
                             {
                                 value = resourceLoader.GetString("ZeroLength");
                             }
+                            else
```

Comment:
Is REG_NONE ever supported for more than an empty string?

Reply:
`REG_NONE` is basically an undefined value type but behaves the same as a `REG_BINARY`.

Reply:
That's actually incorrect: per the Registry documentation, it's not supposed to have a value of any kind.

https://learn.microsoft.com/en-us/windows/win32/sysinfo/registry-value-types calls out: REG_NONE as having "No defined value type."

Reply:
That's what I first thought too until I found out that a `REG_NONE` can be edited just like a `REG_BINARY` in Regedit. Not exactly sure why Regedit allows that.

![image](https://github.com/microsoft/PowerToys/assets/3975545/e38d8694-8375-4881-a4b3-a7265e266ae2)

![image](https://github.com/microsoft/PowerToys/assets/3975545/48aabc55-8152-4e33-a6b1-680be48815f3)

Reply:
Weird.